Dockerfile

```diff
@@ -0,0 +1,94 @@
+ARG UBUNTU_VERSION=18.04
+FROM nvidia/cuda:10.0-cudnn7-runtime-ubuntu${UBUNTU_VERSION}
+# python3.6 is provided by ubuntu18.04
+# python3.5 is provided by ubuntu16.04
+
+MAINTAINER blueoil-admin@leapmind.io
+
+# TensorBoard
+EXPOSE 6006
+
+ENV LD_LIBRARY_PATH /usr/local/cuda/extras/CUPTI/lib64:${LD_LIBRARY_PATH}
+ENV DEBIAN_FRONTEND noninteractive
+
+# Install basic packages
+RUN apt-get update && apt-get install -y \
+    cmake \
+    locales\
+    python3 \
+    python3-dev \
+    python3-pip \
+    python3-wheel
+
+# Locale setting
+RUN locale-gen en_US.UTF-8
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US:en
+ENV LC_ALL en_US.UTF-8
+
+RUN ln -s /usr/bin/python3 /usr/bin/python
+RUN ln -s /usr/bin/pip3 /usr/bin/pip
+
+# Install dependencies for Pillow, Scipy and matplotlib for display
+RUN apt-get install -y --no-install-recommends \
```

Comment:
I think it is better to merge this to [line:15](https://github.com/blue-oil/blueoil/pull/374/files#diff-3254677a7917c6c01f55212f86c57fbfR15).
Please change to run `apt-get update` and `apt-get clean && rm -rf /var/lib/apt/lists/*` only once to reduce build time and layer size.

Reply:
?? This PR is to reduce `apt update` and `apt clean`. But I guess we can merge apt lines as you mention except `locale` settings